src/Validation/Validation.php

```diff
@@ -658,7 +658,7 @@ public static function localizedTime($check, string $type = 'datetime', $format
         if ($check instanceof DateTimeInterface) {
             return true;
         }
-        if (is_object($check)) {
+        if (!is_string($check)) {
```

Comment:
It would be good to expand the test cases for this method to cover this scenario. Do you need a hand with that?

Reply:
I added some test cases.